src/serialization/protoBufTest.py

```diff
@@ -0,0 +1,149 @@
+import pickle
+import sys
+import numpy as np
+import font_pb2
+
+######################################################
+# FOR TESTING THE FACT THAT PROTBUF AND PICKLE MATCH #
+######################################################
```

Comment:
@ostapstephan what does this script do? Does it validate that the protobuf and pickle data files contain the same data? It looks like there's an awful lot of duplicated code between this file and `pkl2proto.py`: why?

I just find it weird that I'm writing code that goes `from protoBufTest import readSerializedProto`.

Reply:
cc @ostapstephan